src/main/java/frc/robot/Robot.java

```diff
@@ -22,12 +36,11 @@ public void disabledPeriodic() {
 
     @Override
     public void teleopInit() {
-
+        //new IntakeCommandAxes(intakeSystem, xboxController).schedule();
```

Comment:
delete this line. No need for it since we don't use this command anymore

Reply:
done